app.py

```diff
@@ -174,13 +208,28 @@ def prepare_metadata():
     sort_image_captions()
 
 
-# Deletes all files uploaded through the GUI and removes them from the dict
-def clean_up():
+# Deletes files uploaded through the GUI and removes them from the dict
+# If 'self' is given then only the current user's images are deleted
+def clean_up_user_images(self):
+    img_prefix = get_user_img_prefix(self) if self else temp_img_prefix
+    img_list = get_image_list()
+    for img_file in img_list:
+        if img_file.startswith(static_img_path + img_prefix):
+            os.remove(img_file)
+            image_captions.pop(img_file)
+
+
+# Deletes expired user uploaded files and removes them from the dict
+# User uploaded images expire after one day
+def clean_up_old_images():
     img_list = get_image_list()
+    exp_time = time.time() - (24 * 60 * 60)
     for img_file in img_list:
-        if img_file.startswith(static_img_path + temp_img_prefix):
+        if img_file.startswith(static_img_path + temp_img_prefix)\
```

Comment:
nit, i don't like using slashes, you can use parens:

```
        if (img_file.startswith(static_img_path + temp_img_prefix)
                and os.stat(img_file).st_ctime < exp_time):
```

Reply:
thank you, I couldn't remember how to get it to work with out the `slash` that my IDE kept adding.